reference/constraints/File.rst

```diff
@@ -242,6 +242,17 @@ of given mime types (if an array).
 
 You can find a list of existing mime types on the `IANA website`_.
 
+.. note::
+
+    When using this constraint on a :doc:`FileType form field </reference/forms/types/file>`,
+    the value of the ``mimeTypes`` option will be used to automatically set the
```

Comment:
We should make clear, that "automatically" only applies if the `FileType` wasn't set (i.e. that `null` was passed as the type or the type argument was completely omitted). Otherwise, the form type guessing would not happen at all.